packages/test-utils/src/testUtils/appWrappers.tsx

```diff
@@ -228,7 +233,7 @@ export function wrapInTestApp(
  * @public
  */
 export async function renderInTestApp(
```

Comment:
This needs to be updated to use `createRoot` as well, right?

Getting the following warning for all our frontend tests using `renderInTestApp`:
```
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
```

Reply:
Missed the `TODO` comment above. Oops. 😅